### DISCUSSION TITLE
EMAN2 Import

### DISCUSSION THREAD

**Discussion by jakecroft** [nextpyp/.github#117] [Q&A] [answered]
Is it possible to import an EMAN2 project into nextPYP? I know the transformations for the tomogram reconstruction are specified differently in EMAN2 than other reconstruction software, but it would be nice to import an eman2 project without having to redo reconstruction and re-pick particles.

**Comment by abartesaghi** [comment, marked as answer]
We don't currently have a way to import eman2 projects into nextPYP. For importing tilt-series/tomograms, nextPYP can import metadata in the IMOD format (.tlt + .xf files), so it would be a matter of coverting the eman metadata into the imod format. To import particle locations, one would first need to make sure the tomographic reconstructions align perfectly with each other. In that case, the eman particle coordinates would need to be converted into IMOD's txt format (as produced by IMOD's model2point) and imported into nextPYP during the pre-processing block.
